### PR TITLE
Be more precise when refuse to expire

### PR DESCRIPTION
**Pull request by VicDeo**
Change the wording to be more clear

## Description
Change the wording to be more clear

## Related Issue
https://github.com/owncloud/core/issues/28509

## Motivation and Context
Unclear output

## How Has This Been Tested?
By reading the message

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tomneedham** [comment]
Text looks good, not tested the code as it is just a string change 👍

**Comment by DeepDiver1975** [comment]
Please add tests. Use command Tester. See other command tests like app commands

**Comment by VicDeo** [comment]
@DeepDiver1975 test that `"Auto expiration is configured - expiration will be handled automatically."` is actually printed char by char?

**Comment by PVince81** [comment]
@VicDeo I usually use `assertContains` with a few keywords. Maybe there was no actual test at all for that code path so adding one would be nice.

**Comment by DeepDiver1975** [comment]
example: https://github.com/owncloud/core/pull/24551/files#diff-53868c402aa661f06b79e3055d46e383

**Comment by VicDeo** [comment]
Covered

File: apps/files_trashbin/tests/Command/ExpireTrashTest.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+use OCA\Files_Trashbin\Command\ExpireTrash;
+use OCA\Files_Trashbin\Expiration;
+use OCP\IUserManager;
+use Symfony\Component\Console\Tester\CommandTester;
+use Test\TestCase;
+
+
+/**
+ * Class ExpireTrashTest
```
Comment (PVince81) [inline apps/files_trashbin/tests/Command/ExpireTrashTest.php]: uh oh, class name mismatch

**Review by PVince81 — APPROVED** [review]
:+1: fix the phpdoc and this is good to go

**Comment by VicDeo** [comment]
@PVince81 fixed, history squashed, waiting for CI

**Comment by VicDeo** [comment]
Stable10: https://github.com/owncloud/core/pull/29671

@PVince81 what about stable9.1?

**Comment by PVince81** [comment]
If not critical I'd say no need to backport for 9.1.

**Pull request merged**